webrtc-stats.html

```diff
@@ -622,8 +622,43 @@ <h3>
           the first RTCP packet is sent or received that refers to the <a>SSRC</a> of the <a>RTP stream</a>.
         </p>
         <p>
-          RTP monitored objects are not [= deleted =].
+          RTP monitored objects are deleted when the corresponding RTP sender or
+          RTP receiver is reconfigured to remove the corresponding RTP stream.
+          This happens for the old SSRC when the {{RTCRtpStreamStats/ssrc}}
+          changes, a simulcast layer is dropped or the {{RTCRtpTransceiver}}'s
+          {{RTCRtpTransceiver/currentDirection}} becomes <code>"stopped"</code>.
+          The monitored object is <i>not</i> deleted if the transceiver is made
+          <code>"inactive"</code> or if the encoding parameters'
+          {{RTCRtpEncodingParameters/active}} attribute is set to
+          <code>false</code>. If an SSRC is recycled after a deletion event has
+          happened, this is considered a new RTP monitored object and the new 
+          RTP stream stats will have reset counters and a new ID.
         </p>
+        <p>
+          For a given RTP stats object its total counters must always increase,
+          but due to changes in SSRC, simulast layers dropping or transceivers
+          stopping, an RTP stats object can be deleted and/or replaced by a new
+          RTP stats object. The caller should be aware of this when aggregating
```

Comment:
Good to avoid "should"
```suggestion
          RTP stats object. The caller will need to be aware of this when aggregating
```